css/index.css

```diff
@@ -1,3 +1,383 @@
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: 'Titillium Web', sans-serif;
+}
+/*mobile */
+@media (max-width: 500px) {
+  html {
+    overflow-y: scroll;
+    overflow-x: hidden;
+  }
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  margin: auto;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  .container {
+    width: 93%;
+  }
+}
+
+/* Header Styles */
+
+header {
+  width: 100%;
+  display: flex;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  header {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+}
+
+header nav {
+  margin-top: 40px;
+  width: 610px;
+  justify-content: space-between;
+  display: flex;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  header nav {
+    width: 100%;
+    margin-top: 20px;
+  }
+}
+
+/* mobile */
+@media (max-width: 500px) {
+  header nav {
+    margin-top: 0;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+  }
+}
+
+header nav a {
+  text-decoration: none;
+  color: #b0b0b0;
+}
+
+/* mobile */
+@media (max-width: 500px) {
+  header nav a {
+    border-top: 1px solid black;
+    padding: 20px 50%;
+    font-size: 20px;
+  }
+  .last-link {
+    border-bottom: 1px solid black;
+  }
+}
+
+header nav a:hover {
+  text-decoration: underline;
+  color: black;
+}
+
+header .logo {
+  margin: 28px 0 0 91px;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  header .logo {
+    margin: 30px auto;
+  }
+}
+
+/* mobile */
+@media (max-width: 500px) {
+  header .logo {
+    margin: 20px auto;
+  }
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+  display: flex;
+  margin-top: 72px;
+  justify-content: flex-end;
+}
+
+/* mobile */
+@media (max-width: 500px) {
+  .cta {
+    margin-top: 45px;
+  }
+}
+
+.cta .cta-text {
+  font-size: 72px;
+  text-align: center;
+  letter-spacing: 5px;
+  display: flex;
+  justify-content: center;
+  flex-flow: column;
+  align-items: center;
+  margin: 0 82px 0 0;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  .cta .cta-text {
+    margin: 0 auto;
+  }
+}
+
+.cta .cta-text button {
+  border: 1px double black;
+  font-size: 16px;
+  background: white;
+  width: 170px;
+  height: 32px;
+  margin-top: 10px;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+/* tablet */
+@media (max-width: 800px) {
+  .cta img {
+    display: none;
+  }
+}
+
+/* Main Content Styles */
```

Comment:
Way to comment out your sections, makes it really to see which section I'm looking at!